src/admin/resolvers/mutations/RejectedItem.ts

```diff
@@ -23,6 +24,20 @@ export async function createRejectedItem(
     throw new AuthenticationError(ACCESS_DENIED_ERROR);
   }
 
+  // validate reason enum
+
+  // rejection reason comes in as a comma separated string
+  // split them up and strip out preceeding/trailing white space
+  const reasons = data.reason.split(',').map((reason) => {
+    return reason.trim();
```

Comment:
if you wanted to simplify a bit, you could move the for loop logic into this iterator (either using forEach, or map, whichever)